R/export_manifest.R

```diff
@@ -0,0 +1,102 @@
+#' Export manifest file with metadata
+#'
+#' @param manifest_path Path to the manifest file.
+#' @param input_files List of files that are inputs to the workflow.
+#' @param output_files List of files that are outputs from the workflow.
```

Comment:
Are these expected to be *named* lists? If so, good to specify that, if not test/check for it.

Reply:
Docs clarified in [0677b2e](https://github.com/RMI-PACTA/pacta.workflow.utils/pull/3/commits/0677b2eb412e830ab68104331b27d8983fd8e92e)